src/questions/BaseQuestionPage.tsx

```diff
@@ -25,30 +27,36 @@ function BaseQuestionPage({question}: {question: Question}) {
         updateSearchLink(`/search?${searchParams.toString()}`);
     };
 
-    return <div>
-        <div>{question.name}</div>
-        <div>Checkboxes</div>
-        {question.checkboxes.map((checkbox, index) => {
-            return (
-                <li key={index}>
-                    <div className="toppings-list-item">
-                        <div className="left-section">
-                            <input
-                                type="checkbox"
-                                id={`${question.id}-checkbox-${index}`}
-                                name={checkbox}
-                                value={checkbox}
-                                checked={checkedState[index]}
-                                onChange={() => handleOnChange(index)}
-                            />
-                            <label htmlFor={`custom-checkbox-${index}`}>{checkbox}</label>
+    return <Page>
+        <div className="flex flex-col">
+            <h2>How can we help you?</h2>
+            <h3>Please select from the following:</h3>
+            <div className="category">{question.name}</div>
+            <div className="pl-4 pr-4">
+                {question.checkboxes.map((checkbox, index) => {
+                    return (
+                        <div className="mb-4" key={checkbox}>
+                            <label className="cursor-pointer">
+                                <input
+                                    type="checkbox"
+                                    id={`${question.id}-checkbox-${index}`}
+                                    name={checkbox}
+                                    value={checkbox}
+                                    className="mr-4 cursor-pointer"
+                                    checked={checkedState[index]}
+                                    onChange={() => handleOnChange(index)}
+                                />
+                                {checkbox}
+                            </label>
                         </div>
-                    </div>
-                </li>
-            );
-        })}
-        <Link to={"/"}>Back</Link>
-        <Link to={searchLink}>Next</Link>
-    </div>;
+                    );
+                })}
```

Comment:
This can be moved from a block `{ return <div></div> }` to `(<div></div>)`

Reply:
ahh it absolutely can. Feel free to blame oskar for that one

Reply:
If you can that'd be great

Reply:
oh I already fixed that